themes/red-theme/lms/templates/header.html

```diff
@@ -43,7 +43,7 @@
     <h1 class="logo">
       <a href="${marketing_link('ROOT')}">
         <%block name="navigation_logo">
-            <img src="${static.url("images/logo.png")}" alt="${platform_name()}"/>
+            <img src="${static.url("images/logo.png")}" alt="${_("{platform_name} Home Page").format(platform_name=settings.PLATFORM_NAME)}"/>
```

Comment:
Did you test this? I think here you may have to use `platform_name=platform_name()` given the line you're deleting.